docs/python/reference/dbos-class.md

```diff
@@ -75,10 +75,45 @@ Assuming your file is `main.py`, run with `python3 -m main` (dev) or `gunicorn -
 
 ### destroy
 
-```
-DBOS.destroy()
+```python
+DBOS.destroy(
+    destroy_registry: bool = True
+)
 ```
 
 Destroy the DBOS singleton, terminating all active workflows and closing database connections.
 After this completes, the singleton can be re-initialized.
-Useful for testing.
\ No newline at end of file
+Useful for testing.
+
+**Parameters:**
+- `destroy_registry`: Whether to destroy the global registry of decorated functions. If you plan to call `DBOS.launch()` again, you likely want to set this to `False`.
+
+## Configuration Management
+
+### load_config
+
+```python
+load_config(
+    config_file_path: str = "dbos-config.yaml",
+    use_db_wizard: bool = True
+) -> ConfigFile:
+```
+
+Load and parse a DBOS configuration file into a `ConfigFile` object.
+
+**Parameters:**
+- `config_file_path`: The path to the DBOS configuration file to parse.
+- `use_db_wizard`: If the configuration file specifies default database connection parameters and there is no Postgres database there, whether to prompt the user to connect to a different database.
```

Comment:
I don't see the db wizard documented in the [TS docs](https://docs.dbos.dev/typescript/reference/transactapi/dbos-class#dbosgetconfig). Is that something we need to add?